src/main/java/uk/gov/service/notify/NotificationClient.java

```diff
@@ -23,6 +23,29 @@
     private final String baseUrl;
     private final Proxy proxy;
 
+    public NotificationClient(String apiKey, String baseUrl) {
+        this(
+                apiKey.substring(Math.max(0, apiKey.length() - 36)),
+                apiKey.substring(
+                        Math.max(0, apiKey.length() - 73),
+                        Math.max(0, apiKey.length() - 37)
+                ),
+                baseUrl
+        );
```

Comment:
does this function not need to call `setDefaultSSLContext()`, in the same way that `NotificationClient(String apiKey, String serviceId, String baseUrl)` does on line 49?